abacatepay/__init__.py

```diff
@@ -8,45 +8,83 @@
+billing = x = client.create_billing(products=[Product(externalId="123", name="Teste", quantity=1, price=101, description="Teste")], returnURL="https://abacatepay.com", completionUrl="https://abacatepay.com")
 print(billing.data.url)
 # > https://abacatepay.com/pay/aaaaaaa
 ```
 
 More examples found on https://abacatepay.readme.io/
 """
 
-
-from .src.billing import Billing
+import requests
+import logging
 from typing import Literal
-from ._models import Product, IBilling, Customer
+
+from .billing import Billing
+from ._models import Product, Billing, Customer
 from ._constants import BILLING_KINDS, BILLING_METHODS, BASEURL, USERAGENT
 from ._exceptions import *
-import requests
+
+
+logger = logging.getLogger(__name__)
+
 
 class AbacatePay:
-  """
-  The AbacatePay sdk client.
-  """
-
-  def __init__(self, api_key: str):
-    self.api_key = api_key
-
-  def request(self, url: str, method: Literal["GET", "POST", "PUT", "PATCH", "DELETE"] = "GET", **kwargs):
-    return requests.request(method, url,  headers={"Authorization": f"Bearer {self.api_key}", "User-Agent": USERAGENT}, **kwargs)
-
-  def CreateBilling(self, products:list[Product], returnURL:str, completionUrl:str, methods:list[BILLING_METHODS]=['PIX'], frequency:BILLING_KINDS='ONE_TIME', customerId:str|None=None, customer:Customer|None=None):
-    return Billing(products, returnURL, completionUrl, self.api_key, methods, frequency=frequency, customerId=customerId, customer=customer)
-
-  def listBills(self) -> list[IBilling]:
-    response = self.request(f"{BASEURL}/billing/list", method="GET")
-    try:
-        if response.status_code == 200:
-            return [IBilling(data=bill) for bill in response.json()['data']]
-        else:
-            raise_for_status(response)
-    except requests.exceptions.Timeout:
-        raise APITimeoutError(request=response)
-    except requests.exceptions.ConnectionError:
-        raise APIConnectionError(message="Connection error.", request=response)
+    """
+    The AbacatePay sdk client.
+    """
+
+    def __init__(self, api_key: str):
+        self.api_key = api_key
+
+    def _request(
+        self,
+        url: str,
+        method: Literal["GET", "POST", "PUT", "PATCH", "DELETE"] = "GET",
+        **kwargs,
+    ):
+        return requests.request(
+            method,
+            url,
+            headers={
+                "Authorization": f"Bearer {self.api_key}",
+                "User-Agent": USERAGENT,
+            },
+            **kwargs,
+        )
+
+    def create_billing(
+        self,
+        products: list[Product],
+        returnURL: str,
+        completionUrl: str,
+        methods: list[BILLING_METHODS] = ["PIX"],
+        frequency: BILLING_KINDS = "ONE_TIME",
+        customerId: str | None = None,
+        customer: Customer | None = None,
+    ):
+        return Billing(
+            products,
+            returnURL,
+            completionUrl,
+            self.api_key,
```

Comment:
This var is now private, so now it's called __api_key

See https://github.com/AbacatePay/abacatepay-python-sdk/pull/2/commits/5fab0da4942c2ad36417eebc40e8bb60354f9f38